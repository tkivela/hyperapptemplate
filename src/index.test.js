import { app } from 'hyperapp'
import { render } from 'hyperapp-render'

import actions from './actions'
import state from './state'
import view from './View'

test('view to match snapshot', () => {
  const tree = render(app)(state, actions, view)
  expect(tree.toString()).toMatchSnapshot()
})
