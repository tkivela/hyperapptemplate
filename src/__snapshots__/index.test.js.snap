// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`view to match snapshot 1`] = `"<div __source=\\"[object Object]\\"><h1 __source=\\"[object Object]\\">Count: 0</h1><button style=\\"color:red;height:30px;font-size:24px;line-height:0\\" __source=\\"[object Object]\\">-</button><button style=\\"color:lightgreen;height:30px;font-size:24px;line-height:0\\" __source=\\"[object Object]\\">+</button></div>"`;
